tools_docker/Debian_Uwsgi/entrypoint.sh

```diff
@@ -113,6 +113,25 @@ else
     sed -i "s|^\s*ProxyPassReverse /agenda http://127.0.0.1:8080/agenda|    #ProxyPassReverse /agenda http://127.0.0.1:8080/agenda|g" /etc/tracim/apache2.conf
 fi
 
+# Create-index or Update-index for elasticsearch
+export SEARCH__ENGINE_TYPE=$(cat /etc/tracim/development.ini |grep "search.engine = " | cut -c17-)
```

Comment:
create an issue for making this search more resilient (ie using a regular expression instead of grep/cut)

Reply:
Done